Mathlib/Data/Matroid/Rank/Cardinal.lean

```diff
@@ -342,4 +356,34 @@ instance invariantCardinalRank_comap (M : Matroid β) [InvariantCardinalRank M]
 
 end Instances
 
```

Comment:
Maybe these new lemmas are more naturally placed *before* the `InvariantCardinalRank` definition?